homeassistant/components/roomba/config_flow.py

```diff
@@ -57,34 +54,156 @@ class RoombaConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
 
+    def __init__(self):
+        """Initialize the roomba flow."""
+        self.discovered_robots = {}
+        self.name = None
+        self.blid = None
+        self.host = None
+
     @staticmethod
     @callback
     def async_get_options_flow(config_entry):
         """Get the options flow for this handler."""
         return OptionsFlowHandler(config_entry)
 
-    async def async_step_import(self, import_info):
-        """Set the config entry up from yaml."""
-        return await self.async_step_user(import_info)
-
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
+        # This is for backwards compatibility.
```

Comment:
I don't think there's any backwards compatibility needed here. The user step is enough.